test/test_reductions.py

```diff
@@ -1129,6 +1129,10 @@ def test_bincount(self, device):
             torch.bincount(torch.tensor([1, 3], device=device),
                            torch.tensor([.2, .2], device=device),
                            minlength=-1)
+        # n-d weights, with n > 1 throws
```

Comment:
We should probably port these assertions to ErrorInputs on the bincount OpInfo, but this is fine for now